feature_extraction.py

```diff
@@ -140,3 +140,215 @@ def transform(self, X):
                         how='left')
 
         return {'numerical_features': X[self.groupby_aggregations_names].astype(np.float32)}
+
+
+class Application(BaseTransformer):
+    def __init__(self):
+        super().__init__()
+
+    @property
+    def application_names(self):
```

Comment:
@pknut If there is no calculation in the method why not have 

`python
self.application_names = ['A', 'B']
`